src/ybus_calculations.jl

```diff
@@ -7,16 +7,27 @@ struct Ybus{Ax, L <: NTuple{2, Dict}} <: PowerNetworkMatrix{ComplexF64}
     data::SparseArrays.SparseMatrixCSC{ComplexF64, Int}
     axes::Ax
     lookup::L
+    data_ft::SparseArrays.SparseMatrixCSC{ComplexF64, Int}
+    data_tf::SparseArrays.SparseMatrixCSC{ComplexF64, Int}
```

Comment:
Do these `data_ft` and `data_tf` matrices always have just one nonzero entry per row?

Reply:
Resolved.